### DISCUSSION TITLE
GitHub API call to detect whether a github org is part of an enterprise org?

### DISCUSSION THREAD

**Discussion by caniszczyk** [todogroup/ospology#73] [OSPO Discussions]
https://github.com/containerd is part of a GHE https://github.com/enterprises/cncf

Is there an easy way to detect this via GitHub API call whether an org is owned by a GHE account?

**Comment by geekygirldawn** [comment]
I think this is in the GraphQL GitHub API, and I think you can only access the info if you are an org owner and have given your API key access to this type of org info.

**Reply by geekygirldawn** [reply]
https://docs.github.com/en/graphql/reference/objects#enterprise will give you the orgs that belong to that enterprise. I'm not sure if it works the other way around?

**Reply by byjrack** [reply]
Im know on my private Enterprise orgs and on public containerd you can see the "Part of <ent>" next to the octicon org SVG in the HTML so feel s like there is some api that contains that attribute to make rendering easy

**Reply by geekygirldawn** [reply]
Ha, you would think so, but there are loads of things visible via the html interface that are not in the API. [Dependents](https://github.com/vmware-tanzu/velero/network/dependents), for example, is not available in the API, but [Dependencies](https://github.com/vmware-tanzu/velero/network/dependencies) are in the API. I've filed a request to get dependents added 🤞

**Reply by byjrack** [reply]
Yup and as I thought it about it more the Org API is usually org:admin for any meaningful data so even if it was there it's really just useful for your Orgs.  So not sure if @caniszczyk is looking for something for the CNCF or for affiliation info for other parties.  Feels like a HTTP GET and parse may be the easiest and should be somewhat reliable.
